addons/spreadsheet/static/src/list/list_data_source.js

```diff
@@ -163,6 +165,16 @@ export default class ListDataSource extends OdooViewsDataSource {
     // Private
     //--------------------------------------------------------------------------
 
+    _formatDateTime(dateValue) {
+        const date = parseDateTime(dateValue);
```

Comment:
marche pô :cry: 
It parses the datetime as if it was local
```js
const date = parseDateTime('2023-02-21 13:02:58')
date.toISO()
> '2023-02-21T13:02:58.000+01:00'
```
I think `deserializeDate` should be used